build.gradle

```diff
@@ -166,25 +192,39 @@ static void assertExecutableSpecRequirementsForTests(Project project, Test test)
     test.systemProperties['concordion.output.dir'] = project.reporting.baseDir.getAbsolutePath() + "/spec"
     test.systemProperties['concordion.run.threadCount'] = '2.5C' // 2.5 * number of cores
 
-    def propertiesToMap = ['CT_PROJECT_KEY',
-                           'CT_CLIENT_ID',
-                           'CT_CLIENT_SECRET',
-                           'CT_PAYONE_INTEGRATION_URL',
+    def propertiesToMap = ['TEST_DATA_CT_PROJECT_KEY',
```

Comment:
because these values are mapped from `gradle.properties`, but the tests should be independent from build tool.